tracer/src/Datadog.Trace.ClrProfiler.Managed.Loader/Startup.NetCore.cs

```diff
@@ -90,15 +97,16 @@ private static Assembly ResolveAssembly(string name)
                     return cachedAssembly;
                 }
 
-                // Only load the main profiler into the default Assembly Load Context.
-                // If Datadog.Trace or other libraries are provided by the NuGet package their loads are handled in the following two ways.
-                // 1) The AssemblyVersion is greater than or equal to the version used by Datadog.Trace, the assembly
+                // Only load the main profiler into the default AssemblyLoadContext.
+                // If the NuGet package provides Datadog.Trace or other libraries, loading them is handled in the following two ways:
+                // 1) If the AssemblyVersion is greater than or equal to the version used by Datadog.Trace, the assembly
                 //    will load successfully and will not invoke this resolve event.
-                // 2) The AssemblyVersion is lower than the version used by Datadog.Trace, the assembly will fail to load
+                // 2) If the AssemblyVersion is lower than the version used by Datadog.Trace, the assembly will fail to load
                 //    and invoke this resolve event. It must be loaded in a separate AssemblyLoadContext since the application will only
-                //    load the originally referenced version
-                StartupLogger.Debug("Loading {0} with DependencyLoadContext.LoadFromAssemblyPath", path);
-                var assembly = DependencyLoadContext.LoadFromAssemblyPath(path); // Load unresolved framework and third-party dependencies into a custom Assembly Load Context
+                //    load the originally referenced version.
+                StartupLogger.Debug("Calling DependencyLoadContext.LoadFromAssemblyPath(\"{0}\")", path);
+                var assembly = DependencyLoadContext.LoadFromAssemblyPath(path); // Load unresolved framework and third-party dependencies into a custom AssemblyLoadContext
+                StartupLogger.Debug("Assembly loaded: {0}", assembly.FullName);
```

Comment:
Do we really need this? `assembly.FullName` is not cached and internally is a QCall. https://source.dot.net/#System.Private.CoreLib/src/System/Reflection/RuntimeAssembly.cs,142

If is not necessary let's remove it.

Reply:
This was just there from before. Removed, thanks!